lms/views/email.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from h_pyramid_sentry import report_exception
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import view_config
+
+from lms.services import EmailUnsubscribeService
+from lms.services.exceptions import ExpiredJWTError, InvalidJWTError
+
+LOG = logging.getLogger(__name__)
+
+
+@view_config(
+    route_name="email.unsubscribe",
+    request_method="GET",
+    renderer="lms:templates/message.html.jinja2",
```

Comment:
When this view is successful it actually returns a redirect rather than using this renderer. I think you might want to omit the renderer from the `@view_config` and only specify it with a `request.override_renderer = "..."` in the `except` block